datasets/penn_treebank/README.md

```diff
@@ -0,0 +1,150 @@
+annotations_creators:
+- expert-generated
+language_creators:
+- found
+languages:
+- en
+licenses:
+- other-LDC User Agreement for Non-Members
+multilinguality:
+- monolingual
+size_categories:
+- n<1K
+source_datasets:
+- original
+task_categories:
+- structure-prediction
+task_ids:
+- structure-prediction-other-POS Tagging
```

Comment:
this should be sequence modeling and language modeling since there aren't any POS tags

Reply:
updated